edx_solutions_api_integration/groups/tests.py

```diff
@@ -1036,3 +1036,12 @@ def test_groups_groups_detail_invalid_group_id(self):
         response = self.do_get(test_uri)
         self.assertEqual(response.status_code, 404)
 
+    def test_groups_courses_list_missing_course_id(self):
+
+        # Create test group
+        test_group = GroupFactory.create()
+
+        # Test with missing course_id in the request data
+        test_uri = '{}/{}/courses/'.format(self.base_groups_uri, test_group.id)
+        response = self.do_post(test_uri, {})
```

Comment:
Since we are testing for absence of course_id we shoud have data dict like this ` {"course_id": ""}`